code/modules/hydroponics/grown_inedible.dm

```diff
@@ -151,17 +154,35 @@
 	if(!..())
 		return
 	if(istype(M, /mob/living))
-		to_chat(M, "<span class='warning'>You are heated by the warmth of the of the [name]!</span>")
-		M.bodytemperature += potency/2 * TEMPERATURE_DAMAGE_COEFFICIENT
+		to_chat(M, "<span class='warning'>You are [tampermessage] the of the [name]!</span>")
+		M.bodytemperature += (hotorcold * potency)/2 * TEMPERATURE_DAMAGE_COEFFICIENT
```

Comment:
hmmm, -150C body temp adjustment at 200 potency
i wonder how this will combine with hinaichigo's reagent heating system and what happens once you hit someone 3 times and go below below absolute zero

Reply:
Just tested it, you go under absolute zero, there are no checks for that level of cold. Not sure where to add said check.
It is about as deadly as a novaflower at the same potency, though I can reduce the cold to a quarter of the potency instead of a half.